### PR TITLE
lightning: update the default value of add-index-by-sql to false

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

as tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/13391

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ addr = "172.16.31.10:8287"
 # 默认值为 MaxInt64 字节，即 9223372036854775807 字节。
 # disk-quota = "10GB" 
 
-# Physical Import Mode 是否通过 SQL 方式添加索引。默认根据 TiDB 版本自动选择。如果 TiDB 版本早于 v7.0.0，则默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。从 v7.0.0 版本开始，默认为 `true`，即 TiDB Lightning 会在导入数据完成后，使用 add index 的 SQL 来添加索引。
+# Physical Import Mode 是否通过 SQL 方式添加索引。则默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。如果设置为 `true`，即 TiDB Lightning 会在导入数据完成后，使用 add index 的 SQL 来添加索引。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-configuration.md:149]:
```suggestion
# Physical Import Mode 是否通过 SQL 方式添加索引。默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。如果设置为 `true`，即 TiDB Lightning 会在导入数据完成后，使用 add index 的 SQL 来添加索引。
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 2]
```diff
@@ -28,7 +26,7 @@ Physical Import Mode 对应的后端模式为 `local`。
 
     引擎文件包含两种：**数据引擎**与**索引引擎**，各自又对应两种键值对：行数据和次级索引。通常行数据在数据源里是完全有序的，而次级索引是无序的。因此，数据引擎文件在对应区块写入完成后会被立即上传，而所有的索引引擎文件只有在整张表所有区块编码完成后才会执行导入。 
     
-    注意当 `tidb-lightning` 使用 SQL 接口添加索引时，索引引擎将不会写入数据，因为此时目标表的次级索引已经在第 2 步中被移除。
+    注意当 `tidb-lightning` 使用 SQL 接口添加索引时（即 `add-index-by-sql` 设置为 `true`），索引引擎将不会写入数据，因为此时目标表的次级索引已经在第 2 步中被移除。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:29]: 第 2 步仍然需要有关于这个选项的描述。

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 3]
```diff
@@ -18,8 +18,6 @@ Physical Import Mode 对应的后端模式为 `local`。
 
 2. `tidb-lightning` 在目标数据库建立表结构，并获取其元数据。
```
[the remark this answers is not in the document]
Comment (gozssky) [reply tidb-lightning/tidb-lightning-physical-import-mode.md:20]: LGTM

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -146,7 +146,7 @@ addr = "172.16.31.10:8287"
 # 默认值为 MaxInt64 字节，即 9223372036854775807 字节。
 # disk-quota = "10GB" 
 
-# Physical Import Mode 是否通过 SQL 方式添加索引。默认根据 TiDB 版本自动选择。如果 TiDB 版本早于 v7.0.0，则默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。从 v7.0.0 版本开始，默认为 `true`，即 TiDB Lightning 会在导入数据完成后，使用 add index 的 SQL 来添加索引。
+# Physical Import Mode 是否通过 SQL 方式添加索引。默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。如果设置为 `true`，即 TiDB Lightning 会在导入数据完成后，使用 add index 的 SQL 来添加索引。
 # 通过 SQL 方式添加索引的优点是将导入数据与导入索引分开，可以快速导入数据，即使导入数据后，索引添加失败，也不会影响数据的一致性。
 # add-index-by-sql = true
```
Comment (dsdashun) [inline tidb-lightning/tidb-lightning-configuration.md:152]:
```suggestion

# 在使用 Lightning 导入多租户的 TiDB Cluster 的场景下，指定对应的 key space 名称。默认取值为空字符串，表示 Lightning 会去自动获取这次导入对应租户的 key space 名称；如果指定了值，那么使用指定的 key space 名称来导入。
# keyspace-name = ""
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -146,10 +146,12 @@ addr = "172.16.31.10:8287"
...
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:151]:
```suggestion
# add-index-by-sql = false
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 6]
```diff
@@ -40,7 +40,7 @@ sorted-kv-dir = "./some-dir"
 # 限制 TiDB Lightning 向每个 TiKV 节点写入的带宽大小，默认为 0，表示不限制。
 # store-write-bwlimit = "128MiB"
 
-# Physical Import Mode 是否通过 SQL 方式添加索引。默认根据 TiDB 版本自动选择。如果 TiDB 版本早于 v7.0.0，则默认为 `false` 表示仍然会用 TiDB Lightning 将所需导入的数据以及索引编码成 KV pairs 后再一同导入到 TiKV，实现机制和历史版本保持一致。从 v7.0.0 版本开始，默认为 `true`，即 TiDB Lightning 会导完数据后，再使用 add index 的 SQL 来添加索引。
+# Physical Import Mode 是否通过 SQL 方式添加索引。默认为 `false`，表示 TiDB Lightning 会将行数据以及索引数据都编码成 KV pairs 后一同导入 TiKV，实现机制和历史版本保持一致。如果设置为 `true`，即 TiDB Lightning 会导完数据后，再使用 add index 的 SQL 来添加索引。
 # 通过 SQL 方式添加索引的优点是将导入数据与导入索引分开，可以快速导入数据，即使导入数据后，索引添加失败，也不会影响数据的一致性。
 # add-index-by-sql = true
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:45]:
```suggestion
# add-index-by-sql = false
```

**Pull request merged**